gax-grpc/src/main/java/com/google/api/gax/grpc/ChannelPool.java

```diff
@@ -37,19 +38,23 @@
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.concurrent.Executors;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
+import javax.annotation.Nullable;
 
 /**
  * A {@link ManagedChannel} that will send requests round robin via a set of channels.
  *
  * <p>Package-private for internal use.
  */
 class ChannelPool extends ManagedChannel {
+  private static final int EXECUTOR_THREAD_POOL_SIZE = 2;
```

Comment:
Please name something more specific to avoid confusing it with the general executor. Maybe `REFRESH_EXECUTOR_SIZE`